sympy/series/tests/test_limits.py

```diff
@@ -65,6 +65,10 @@ def test_basic1():
     assert limit(1/tan(x**3), x, (2*pi)**Rational(1, 3), dir="-") is -oo
     assert limit(1/cot(x)**3, x, (pi*Rational(3, 2)), dir="+") is -oo
     assert limit(1/cot(x)**3, x, (pi*Rational(3, 2)), dir="-") is oo
+    assert limit(tan(x), x, oo) == AccumBounds(S.NegativeInfinity, S.Infinity)
+    assert limit(cot(x), x, oo) == AccumBounds(S.NegativeInfinity, S.Infinity)
+    assert limit(sec(x), x, oo) == Limit(sec(x), x, oo, dir='-')
```

Comment:
Could this be AccumBounds as well?

Reply:
This has been pointed out before , I'll try to locate where( was a discussion on `AccumBounds` by Aaron ) . As per what I can recollect `limit(sec(x), x, oo)` would be `AccumBounds(-oo, -1).union(AccumBounds(-oo, -1))` but as union method for `Accumbounds` is under discussion(#10390) in we should restrict from returning the wrong answer (which in this case would be `AccumBounds(-oo, oo)` untill then !

Reply:
I've made requested changes in the latest commit ! LGTM and would be glad if you could confirm .

Reply:
Why does AccumBounds not appear here?

Reply:
okay I'll try to convince you again maybe and if you think that AccumBounds could be better I'll do the needful
On master we have the following 
```
>>> limit(csc(x), x, oo)
Limit(csc(x), x, oo, dir='-')
>>> limit(sec(x), x, oo)
nan
```
Now the correct answer for this is `AccumBounds(-oo, -1).Union(AccumBounds(1, oo))` and currently sympy doesn't have any method to represent this union of AccumBound Intervals and the best approach for this is under discussion in the issue reported above, hence the best we could do is return `AccumBounds(-oo, oo)` . As `AccumBounds(-oo, oo)` doesn't completely represent the correct answer and also this being a basic limit where I feel sympy shouldn't leave any doubt of error ......I thought returning self would be a better choice in this place !

But if you think otherwise , I'll do the needful as soon as possible .

Reply:
AccumBounds is not expected to give an exact description of the set of accumulation points. There may be gaps, even big gaps. The union of two AccumBounds is more accurate than a single one but that can probably seldom be made use of. Hence the simple result seems better to me.

Reply:
I have made the requested change in the latest commit ,would be glad to know your thoughts on the same !